src/client/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -74,29 +73,29 @@ internal static bool IsOperatingSystemAccount(IAccount account)
         }
 
         /// <summary>
-        /// Returns true if MSAL can use an embedded web view (browser). 
+        /// Returns <c>true</c> if MSAL can use an embedded web view (web browser).
         /// </summary>
         /// <remarks>
-        /// Currently there are no embedded web views on Mac and Linux. On Windows, app developers or users should install 
-        /// the WebView2 runtime and this property will inform if the runtime is available, see https://aka.ms/msal-net-webview2
+        /// Currently, there are no embedded web views on macOS and Linux. On Windows, the WebView2 runtime is required and this property will reflect the availability of the component.
+        /// Refer to <see href="https://aka.ms/msal-net-webview2">our documentation</see> for additional details.
```

Comment:
This comment and WebView2 wiki aren't up-to-date.

On Windows
- NetFx comes with legacy web view
- Net6 and net core, don't have embedded web view, must use Msal.Desktop + WebView2.
- Net6-windows comes with WebView2 by default.

We also only show WebView2 UI for non-AAD authorities, AAD authorities get legacy web view.